Makefile

```
@@ -1,7 +1,10 @@
-KUTTL_VERSION=0.5.0
-KIND_VERSION=0.8.1
+KUBEADDONS_REPO ?= kubeaddons-enterprise
+TESTING_BRANCH ?= dev
+
+KIND_VERSION ?= 0.8.1
+KUBECONFIG ?= kubeconfig
```

```suggestion
KUBECONFIG = kubeconfig
```
let's not do this, unless we are testing outside the KIND cluster as for today. Can you rename it to `KUBEADDONS_TEST_KUBECONFIG`?

---

reworked it to avoid shooting myself in the foot again (I deleted my own kubeconfig)